kvdb_mgr.go

```diff
@@ -45,6 +46,14 @@ func New(
 
 	if dsInit, exists := datastores[name]; exists {
 		kvdb, err := dsInit(domain, machines, options, errorCB)
+		for _, wrapperOption := range []string{WrapperEnableLog, WrapperEnableQuorumFilter} {
```

Comment:
Shouldn't this be the other way round? First wrap the actual kvdb object by the quorum filter, and then wrap that with log, so that ErrNoQuorum will get logged. 
Also we can define this list of filters as a constant as the order seems important here.

Reply:
right, log should be last, I am going to remove wrappers at init time and see how they are place properly at runtime.